muddy_macros/src/internal.rs

```diff
@@ -53,7 +53,7 @@ pub fn build_obfuscation_mod(
         pub fn decrypt(encrypted: &[u8], nonce: &[u8]) -> String {
             let nonce = Nonce::from_slice(nonce);
             let plaintext = #cipher_ident.decrypt(nonce, encrypted).unwrap();
-            String::from_utf8(plaintext).unwrap()
+            unsafe { String::from_utf8_unchecked(plaintext) }
```

Comment:
Let me think about this a little bit. I'm not sure that the slight bump in performance is worth introducing `unsafe` into the codebase. Thanks for the contributions!

Reply:
`unsafe` is not a ferocious beast. Actually `unwrap` itself is unsafe. And I also encountered panic while decrypting, I am investigating and trying to provide examples.

Reply:
thanks!

Reply:
`unwrap()` is not unsafe, but it can panic. if it's panicking, that means that the bytes could potentially be invalid utf-8, and therefore you should NOT use from_utf8_unchecked.

Reply:
Sorry, my statement was inaccurate. What I mean is, `unwrap` means that only `Ok` is possible, because there should not be any panic based programs. And `unsafe` can be considered safe if the conditions are met. In fact, `unsafe` is an unchecked version of `unwrap`. Moreover, it is impossible for illegal characters to appear here, so unsafe is feasible.

Reply:
unwrap *could* mean that only `Ok` is possible, but in this context, it's the author of the crate not wanting to return a Result because that takes more effort. this pr should definitely *not* be accepted because, as you noted, it is possible for bytes that result in invalid utf-8 when decrypted to be passed into this function. however, if you do want to accept this pr, you should make the decrypt function unsafe because you can cause undefined behavior just by passing in bytes that weren't generated by the program. the safety guarantee is *very* weak here.

Reply:
You are right. I have fixed the code to ensure this type of guarantee.